README.md

```diff
@@ -156,52 +165,35 @@ with the following content:
 }
 ```
 
-#### * Important - Canonical URLs
-The `url` element in a conformance resource is called the canonical URL and used as a
-unique identifier for profiles on the server. The server does not check to
-see if the URL actually resolves to anything, but it does check to see if
-the `url` attribute is present and the value is unique among others loaded
-on the server.
-
 ### Example Resources
-Next create an example resource in the example resources directory
+Next create an example resource in the example resources directory:
 
-`./site_root/source/examples/research-study-example.json`
+`./site_root/input/resources/examples/rs-001.json`
 
 with the following content:
 
 ```json
 {
     "resourceType":"ResearchStudy",
-    "id": "research-study-example",
+    "id": "rs-001",
     "meta": {
-        "profile": ["http://fhir.kids-first.io/StructureDefinition/ResearchStudy"]
+        "profile": ["http://fhir.kids-first.io/StructureDefinition/study"]
     },
     "status": "completed",
     "title": "Study of Ewing's Sarcoma"
 }
 ```
 ### Validate the Model
-
-To validate the conformance resource along with the example resource you just
-created, run the following:
+To validate the resources you just created:
 
 ```
-# Add resource configuration to IG
-fhirmodel add ./site_root/source/resources/StructureDefinition-ResearchStudy.json
-fhirmodel add ./site_root/source/examples/research-study-example.json
-
 # Run publisher to validate
 fhirmodel validate ./site_root/ig.json --publisher_opts='-tx n/a'
```

Comment:
Shouldn't `./site_root/ig.json` be `./site_root/ig.ini`?

Reply:
Ah yes 👍